addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1016,6 +1016,25 @@ options.registry.OptionsTab = options.Class.extend({
             hsl.lightness);
         return ColorpickerWidget.convertRgbaToCSSColor(adjustedGrayRGB.red, adjustedGrayRGB.green, adjustedGrayRGB.blue);
     },
+    /**
+     * @override
+     */
+    async _renderCustomXML(uiFragment) {
+        await this._super(...arguments);
+        const rangeExtraSaturation = uiFragment.querySelector('we-range[data-param="gray-extra-saturation"]');
+        if (rangeExtraSaturation) {
```

Comment:
Let's use better english with `extraSaturationRange`, not `rangeExtraSaturation` :wink: Also for DOM element I like finishing the variable with `El`, but I won't force you to use that convention too if you don't like it.